website/docs/r/cbr_rule.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cbr_rule"
+description: |-
+  Manages cbr_rule.
+subcategory: "Context Based Restrictions"
+---
+
+# ibm_cbr_rule
+
+Provides a resource for cbr_rule. This allows cbr_rule to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_cbr_rule" "cbr_rule" { 
+	description = "this is an example of rule"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your resource.
+
+* `contexts` - (Optional, List) The contexts this rule applies to.
```

Comment:
Please update docs according to resource schema..
contexts is marked as Optional in docs but is required in resource schema..

Can you please cross check all other arguments in both the resources?

Reply:
changed to Required, and checked other arguments